libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -52,6 +52,8 @@ namespace eosio { namespace chain { namespace webassembly {
          void set_resource_limits(account_name account, int64_t ram_bytes, int64_t net_weight, int64_t cpu_weight);
          void get_resource_limits(account_name account, legacy_ptr<int64_t, 8> ram_bytes, legacy_ptr<int64_t, 8> net_weight, legacy_ptr<int64_t, 8> cpu_weight) const;
          void set_resource_limit(account_name, name, int64_t);
+         uint32_t get_wasm_parameters_packed( span<char> packed_parameters ) const;
+         void set_wasm_parameters_packed( span<const char> packed_parameters );
```

Comment:
You didn't register these two host functions.

Reply:
Also, I think it would be cleaner to put these just below `set_kv_parameters_packed`.